deploy/roles/eod/templates/gdm.schemas.j2

```diff
@@ -0,0 +1,130 @@
+<gdmschemafile>
+  <schemalist>
+
+    <schema>
+      <key>chooser/Multicast</key>
+      <signature>b</signature>
+      <default>false</default>
+    </schema>
+    <schema>
+      <key>chooser/MulticastAddr</key>
+      <signature>s</signature>
+      <default>ff02::1</default>
+    </schema>
+
+    <schema>
+      <key>daemon/User</key>
+      <signature>s</signature>
+      <default>gdm</default>
+    </schema>
+    <schema>
+      <key>daemon/Group</key>
+      <signature>s</signature>
+      <default>gdm</default>
+    </schema>
+    <schema>
+      <key>daemon/AutomaticLoginEnable</key>
+      <signature>b</signature>
+      <default>false</default>
+    </schema>
+    <schema>
+      <key>daemon/AutomaticLogin</key>
+      <signature>s</signature>
+      <default></default>
+    </schema>
+    <schema>
+      <key>daemon/TimedLoginEnable</key>
+      <signature>b</signature>
+      <default>false</default>
+    </schema>
+    <schema>
+      <key>daemon/TimedLogin</key>
+      <signature>s</signature>
+      <default></default>
+    </schema>
+    <schema>
+      <key>daemon/TimedLoginDelay</key>
+      <signature>i</signature>
+      <default>30</default>
+    </schema>
+
+    <schema>
+      <key>debug/Enable</key>
+      <signature>b</signature>
+      <default>false</default>
+    </schema>
+
+    <schema>
+      <key>security/DisallowTCP</key>
+      <signature>b</signature>
+      <default>true</default>
+    </schema>
+
+    <schema>
+      <key>greeter/Include</key>
+      <signature>s</signature>
+      <default>{{ eod_entry }}</default>
```

Comment:
@olivermel Please make the variable name more descriptive, something like eod_greeter_include